app-office/kmymoney/kmymoney-9999.ebuild

```diff
@@ -56,13 +56,17 @@ COMMON_DEPEND="
 	$(add_qt_dep qtsvg)
 	$(add_qt_dep qtwidgets)
 	$(add_qt_dep qtxml)
-	app-office/libalkimia:5
+	app-office/libalkimia
 	dev-libs/gmp:0=
 	dev-libs/libgpg-error
 	x11-misc/shared-mime-info
 	calendar? ( dev-libs/libical:= )
+	hbci? (
```

Comment:
this will also need a build option, for reference, you can look again at the in-tree ebuild but make sure the build system is still doing it the same way